tests/conftest.py

```diff
@@ -15,6 +16,27 @@
 from loguru import logger
 
 
+class InterceptHandler(logging.Handler):
+    """
+    This class intercepts standard logging messages toward Loguru sinks
+    """
+
```

Comment:
nit: remove whitespace

Reply:
removed

Reply:
It seems like `black` tool suggests to reformat file and add whitespace
![Screenshot 2023-01-29 at 17 22 08](https://user-images.githubusercontent.com/99176494/215336392-df0b2849-d180-4561-b734-a13ce12f67eb.png)

Reply:
removed

Reply:
added again - class level